test/fixtures/sites/config/page.js

```diff
@@ -4,27 +4,29 @@
 const urlParams = new URLSearchParams(window.location.search);
 const ready = urlParams.get('ready') ? JSON.parse(urlParams.get('ready')) : undefined;
 
-if (ready && ready.onEditOptions) {
-  ready.onEditOptions = () => {
-    document.getElementById('configure').innerHTML = 'called';
-  };
-}
+function setup() {
```

Comment:
Do you know why that helped? It doesn't seem to change anything (and we have custom widgets that follow the previous pattern).

Reply:
This way, the setup is called only when the page has loaded and the elements it refers to exist. Previously, there was a race - if callbacks happened too early, elements like `configure` could be accessed before they existed.